apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -220,6 +224,22 @@ defmodule OMG.ChildChain.BlockQueue.Core do
   end
 
   # TODO: consider moving this logic to separate module
+  @doc """
```

Comment:
yep, all that crash are "we need to work on" all others are "meh".

So `account_locked`, `nonce_too_low` (in case not warranted as in the doc) _and anything else unexpected_. So crash and act by default.